server/src/main/java/io/littlehorse/server/streamsimpl/coreprocessors/KafkaStreamsLHDAOImpl.java

```diff
@@ -68,18 +65,13 @@
 @Slf4j
 public class KafkaStreamsLHDAOImpl implements LHDAO {
 
-    private Map<String, NodeRun> nodeRunPuts;
     private Map<String, Variable> variablePuts;
```

Comment:
I was not able to use the `StoredGetable` with the `Variable`. When I made the change a bunch of things started to fail, so I decided to delay the migration for later.